keps/sig-node/4191-split-image-filesystem/README.md

```diff
@@ -154,8 +154,8 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
   - [ ] (R) Minimum Two Week Window for GA e2e tests to prove flake free
 - [ ] (R) Graduation criteria is in place
   - [ ] (R) [all GA Endpoints](https://github.com/kubernetes/community/pull/1806) must be hit by [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md)
-- [ ] (R) Production readiness review completed
-- [ ] (R) Production readiness review approved
+- [] (R) Production readiness review completed
```

Comment:
Should it be marked as `[x]` or it's not done yet?

Reply:
I heard that PRR reviews aren't for alpha features. There was a little review of it during the initial phase but I think we should probably consider it for beta once the implementation is complete in the ecosystem.